src/blueprint/from_config/definitions.rs

```diff
@@ -293,13 +293,28 @@ fn validate_field_type_exist(config: &Config, field: &Field) -> Valid<(), String
   }
 }
 
-fn to_fields(type_of: &config::Type, config: &Config) -> Valid<Vec<FieldDefinition>, String> {
+pub fn to_fields(type_of: &config::Type, config: &Config) -> Valid<Vec<FieldDefinition>, String> {
   let to_field = |name: &String, field: &Field| {
     let directives = field.resolvable_directives();
     if directives.len() > 1 {
       return Valid::fail(format!("Multiple resolvers detected [{}]", directives.join(", ")));
     }
 
+    update_args()
+      .and(update_http().trace("@http"))
+      .and(update_unsafe().trace("@unsafe"))
+      .and(update_const_field().trace("@const"))
+      .and(update_modify().trace("@modify"))
+      .and(update_ref_field())
+      .try_fold(&(config, field, type_of, name), FieldDefinition::default())
+  };
+
+  let add_to_field = |name: &String, field: &Field| {
```

Comment:
this closure is literally the same closure as to_field from above. Could we have just one closure with maybe additional argument to do this?